cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -413,23 +416,38 @@ class HydratorPlusPlusConfigStore {
     return this.getConfig().properties;
   }
   setProperties(properties) {
+    let numExecutorKey = 'system.spark.spark.executor.instances';
+    let numExecutorOldKey = 'system.spark.spark.master';
+    let backPressureKey = 'system.spark.spark.streaming.backpressure.enabled';
```

Comment:
nit: all these constants should be `const`. Also, since we're reusing them maybe just define them in one place?